scripts/get-buildconfig.ps1

```diff
@@ -90,6 +90,28 @@ if ("" -eq $Tls) {
         $Tls = "schannel"
     } else {
         $Tls = "openssl"
+        try {
+            # If no Tls was specified, try to guess it based on default OpenSSL version
+            # This is more complicated in attempt to silently deal with missing openssl executable
+            #
+            $pinfo = New-Object System.Diagnostics.ProcessStartInfo
+            $pinfo.FileName = "openssl"
+            $pinfo.Arguments = "version"
+            $pinfo.RedirectStandardError = $true
+            $pinfo.RedirectStandardOutput = $true
+            $pinfo.UseShellExecute = $false
+
+            $p = New-Object System.Diagnostics.Process
+            $p.StartInfo = $pinfo
+            $p.Start() | Out-Null
+            $p.WaitForExit()
+
+            $version = $p.StandardOutput.ReadToEnd()
```

Comment:
Why all this instead of something like:
```PowerShell
$version = & "openssl version"
```

Reply:
I'm no PWSH expert. This gives me 
```
The term 'openssl version' is not recognized as a name of a cmdlet, function, script file, or executable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

Basically what I'm trying to get is silent output when `openssl` binary is missing.

Reply:
pwsh -v
PowerShell 7.3.2

Ubuntu 22.04